ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/persistence/PostgresAround.scala

```diff
@@ -7,15 +7,15 @@ import java.io.StringWriter
 import java.net.ServerSocket
 import java.nio.charset.StandardCharsets
 import java.nio.file.{Files, Path, Paths}
-import java.time.Instant
+import java.util.concurrent.atomic.AtomicBoolean
 
 import com.digitalasset.daml.bazeltools.BazelRunfiles._
 import com.digitalasset.ledger.api.testing.utils.Resource
 import org.apache.commons.io.{FileUtils, IOUtils}
 import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
 import org.slf4j.LoggerFactory
 
-import scala.io.Source
+import scala.collection.JavaConverters._
```

Comment:
If you can, I'd suggest being more granular in importing implicit conversions. I'm very happy to help with this if you need.